src/pocket/PocketALBApplication.spec.ts

```diff
@@ -319,6 +319,21 @@ describe('PocketALBApplication', () => {
     expect(synthed).toMatchSnapshot();
   });
 
+  it('renders an Pocket App with code deploy notifications set to failed only', () => {
```

Comment:
Same as the ECS service test, for this test I'm using the same snapshot as the test above.